static/app/views/metrics/pageHeaderActions.tsx

```diff
@@ -146,12 +149,7 @@ export function PageHeaderActions({showAddMetricButton, addCustomMetric}: Props)
         (hasCustomMetricsExtractionRules(organization) ? (
           <Button
             priority="primary"
-            onClick={() =>
-              navigateTo(
-                `/settings/projects/:projectId/metrics/configure-metric/`,
-                router
-              )
-            }
+            onClick={() => openExtractionRuleCreateModal({}, {onClose: refetch})}
```

Comment:
Just commenting so it is not missed in the follow up, if possible, here also a refetch should only happen if the new metric is created